Makefile

```diff
@@ -108,6 +108,9 @@ benchFrontendBattle:
 benchCrawl:
 	dist/build/LambdaHack/LambdaHack --dbgMsgSer --logPriority 4 --newGame 1 --noAnim --maxFps 100000 --frontendNull --benchmark --stopAfterFrames 7000 --automateAll --keepAutomated --gameMode crawl --setDungeonRng 0 --setMainRng 0
```

Comment:
To let CI pass, all these would need to be changed from "0" to "SMGen 0 0", which is fine as far as I'm concerned.

Reply:
I'd put most bench arguments into `BENCHOPTS=--setDungonRng 0 ...` etc. to reduce duplication. Is that fine?

Reply:
I'd rather be verbose, but trivially copy-pastable from CI logs with errors to commandline, for error replication. Unless `make` inlines some kind of variables?

Reply:
```
% tail Makefile 
FOO=foo bar

foo : 
	echo $(FOO)

% make foo
echo foo bar
foo bar
```

Reply:
Perfect. So BENCHOPTS is fine, because I never copy-paste from the Makefile proper, but always from the output log, to also capture the new random seeds, if generated, and reproduce with them.